docs/apps/components/rate.md

```diff
@@ -0,0 +1,168 @@
+|`this.getValues`|Gets the current values of all fields|`() : Object`|
+|`this.hide`|Hides the field|`(fields: String`&#124;`String[])`|
+|`this.show`|Displays the field|`(fields: String`&#124;`String[])`|
+|`this.disable`| Disable rate field from user interaction|`(fields: String`&#124;`String[])`|
+|`this.enable`| Enable rate field from user interaction|`(fields: String`&#124;`String[])`|
+|`this.setData`|Set the data in the field. The Value object should be of type { componentId: componentValue }|`(Value: Object)`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of the rate component in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+    4. Before using **this.enable(fields)**, make sure the component is **disabled**. This can be done using **this.disable(fields)** or by **enabling** the **Disabled checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/rate1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('rate_2n2vy8wj', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('rate_2n2vy8wj', 'abc')
+        ```
+    
+    3. `this.getValues()`
+        ```js
+        var data = this.getValues();
+        console.log(data);
+        ```
+    
+    4. `this.hide(fields)`
+        ```js
+        var fields= ['rate_2n2vy8wj']
+        this.hide(fields)
+        ```
+    
+    5. `this.show(fields)`
+        ```js
+        var fields= ['rate_2n2vy8wj']
+        this.show(fields)
+        ```
+    
+    6. `this.disable(['fields'])`
+        ```js
+        this.disable(['rate_2n2vy8wj'])
+        ```
+    
+    7. `this.enable(['fields'])`
+        ```js
+        this.enable(['rate_2n2vy8wj'])
+        ```
+    
+    8.  `this.getValue('fieldName')`
```

Comment:
This function is not working correctly. No matter the value of the rate, it is always returning 0. Follow up with Saurabh or Jon and find out if it needs to be removed. 
Also, you have listed this function under * methods *

Reply:
added getValue method